tests/Tests/200__monitor_and_manage/200__metrics/700_monitor.robot

```diff
@@ -0,0 +1,17 @@
+*** Settings ***
+Resource    ../../../Resources/ODS.robot
+Resource    ../../../Resources/Common.robot
+Resource    ../../../Resources/Page/OCPDashboard/OCPDashboard.resource
+Resource    ../../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+Library        JupyterLibrary
+Library        SeleniumLibrary
+Library        OpenShiftCLI
+Library        OperatingSystem
+
+
+*** Test Cases ***
+Verify that Prometheus image is a CPaaS built image with oc
```

Comment:
IMO there's no need to specify `with oc` in the title, since I don't think there are other test cases doing the same in a different way.